Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -6246,5 +6202,49 @@ private MediaAttachment GetMediaAttachment(IReadOnlyList<ResultSetValue> reader)
 
             return item;
         }
+
+#nullable enable
+
+        private readonly struct QueryTimeLogger : IDisposable
+        {
+            private readonly ILogger _logger;
+            private readonly string _methodName;
+            private readonly string _commandText;
+            private readonly Stopwatch? _stopwatch;
+
+            public QueryTimeLogger(ILogger logger, string methodName, string commandText)
+            {
+                _logger = logger;
+                _methodName = methodName;
+                _commandText = commandText;
+                _stopwatch = logger.IsEnabled(LogLevel.Debug) ? Stopwatch.StartNew() : null;
+            }
+
+            public void Dispose()
+            {
+                if (_stopwatch == null)
```

Comment:
```suggestion
                if (_stopwatch is null)
```

You changed all `== null` to `is null` in another pr :)